lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -311,19 +310,23 @@ def filter_resource_file_references(resource_file_references)
           #
           # @return [void]
           #
-          def add_files_to_build_phases(native_target, test_native_targets, app_native_targets)
+          def add_files_to_build_phases(library_native_target, test_native_targets, app_native_targets)
```

Comment:
this is another IDE warning fix. the param is the same as a local variable below.